newm-server/src/main/kotlin/io/newm/server/aws/MdcSetup.kt.kt

```diff
@@ -0,0 +1,32 @@
+package io.newm.server.aws
+
+import org.slf4j.MDC
+import aws.sdk.kotlin.services.ec2.Ec2Client
+import aws.sdk.kotlin.services.ec2.model.DescribeInstancesRequest
+import org.koin.core.qualifier.named
+
+suspend fun setInstanceId() {
+    val instanceId = getInstanceIdFromAws() // Replace with your method to get the instance ID
```

Comment:
What's kotlin approach to run this function on load?

Reply:
In Application.kt, add a new method similar to the other stuff there at the very beginning called `initializeLogging()`. That can call this method I think.